e2e/main.e2e.spec.ts

```diff
@@ -134,16 +135,15 @@ test.describe.serial('Main App Test', () => {
 
 /**
  * Navigate to a specific tab
- * @param path
+ * @param tab
```

Comment:
We should probably describe what `tab` is (expected to be)

Reply:
Fixed.

Reply:
gotcha, fixing it - tks